Library/Homebrew/dev-cmd/tests.rb

```diff
@@ -44,10 +44,10 @@ def tests
       ENV["HOMEBREW_NO_COMPAT"] = "1" if args.no_compat?
       ENV["HOMEBREW_TEST_GENERIC_OS"] = "1" if args.generic?
 
+      ENV.delete("HOMEBREW_NO_GITHUB_API")
```

Comment:
Pedantry but: put this with the other `ENV.delete`s?

Reply:
Also this should still be set when `args.online?` is false.

Reply:
Then I have to move into the spec again, because if it is set, stubbing `GitHub` does not work because it is skipped before it even gets to the point where it calls `search_code`.

Reply:
Seems like it is used for things outside `GitHub` as well. Should be a separate PR in any case.